.github/pull_request_template.md

```diff
@@ -1,26 +1,44 @@
-**Please follow this checklist. Please check each appropriate box (put an 'x' or check it after creating the PR).**
-- [ ] REQUIRED: Do you have an Issue **assigned to you** within the v1 milestone for this PR?  Put the Issue number here:
-- [ ] Provided detailed pull request description and a succinct title (consider template below for guidance).
+<!-- Uncomment sections below as relevant -->
+
+<!-- Title should be a short phrase, e.g. "Add survey functionality". Detailed description should include any design decisions you want reviewers to take note of -->
+
+<!--
+Add the Issue number assigned to you that this PR fully resolves
+Closes #0 
+-->
+
+<!-- 
+#### Screenshots
```

Comment:
I've seen screenshots blow out issues as they can be so large. So I thought it would be better to have this at the end of the PR description. So the other textual parts can be found together

Reply:
Completely agree with the point about screenshots blowing out issues! I've actually set this up to use a `<details>` expander for that exact reason 😄 

It'll look like this:

<details>
<summary>Screenshots</summary>

![example](https://worldhealthorganization.github.io/app/screen-loop.gif)
</details>